app-shells/starship/starship-1.16.0.ebuild

```diff
@@ -0,0 +1,432 @@
+	siphasher@0.3.10
+	slab@0.4.8
+	smallvec@1.10.0
+	socket2@0.4.9
+	starship-battery@0.8.1
+	static_assertions@1.1.0
+	strsim@0.10.0
+	syn@1.0.109
+	syn@2.0.25
+	systemstat@0.2.3
+	tauri-winrt-notification@0.1.1
+	tempfile@3.7.0
+	terminal_size@0.2.6
+	terminfo@0.7.5
+	termios@0.3.3
+	termtree@0.4.1
+	termwiz@0.15.0
+	thiserror@1.0.40
+	thiserror-impl@1.0.40
+	time@0.3.22
+	time-core@0.1.1
+	time-macros@0.2.9
+	tiny-keccak@2.0.2
+	tinyvec@1.6.0
+	tinyvec_macros@0.1.1
+	toml@0.5.11
+	toml@0.7.6
+	toml_datetime@0.6.3
+	toml_edit@0.19.14
+	tracing@0.1.37
+	tracing-attributes@0.1.26
+	tracing-core@0.1.31
+	typenum@1.16.0
+	ucd-trie@0.1.5
+	uds_windows@1.0.2
+	uluru@3.0.0
+	unicase@2.6.0
+	unicode-bidi@0.3.13
+	unicode-bom@2.0.2
+	unicode-ident@1.0.9
+	unicode-normalization@0.1.22
+	unicode-segmentation@1.10.1
+	unicode-width@0.1.10
+	unicode-xid@0.2.4
+	uom@0.34.0
+	url@2.4.0
+	urlencoding@2.1.3
+	utf8parse@0.2.1
+	vcpkg@0.2.15
+	version_check@0.9.4
+	versions@5.0.0
+	vtparse@0.6.2
+	waker-fn@1.1.0
+	walkdir@2.3.3
+	wasi@0.11.0+wasi-snapshot-preview1
+	wasm-bindgen@0.2.87
+	wasm-bindgen-backend@0.2.87
+	wasm-bindgen-macro@0.2.87
+	wasm-bindgen-macro-support@0.2.87
+	wasm-bindgen-shared@0.2.87
+	which@4.4.0
+	winapi@0.3.9
+	winapi-i686-pc-windows-gnu@0.4.0
+	winapi-util@0.1.5
+	winapi-x86_64-pc-windows-gnu@0.4.0
+	windows@0.39.0
+	windows@0.48.0
+	windows-sys@0.48.0
+	windows-targets@0.48.1
+	windows_aarch64_gnullvm@0.48.0
+	windows_aarch64_msvc@0.39.0
+	windows_aarch64_msvc@0.48.0
+	windows_i686_gnu@0.39.0
+	windows_i686_gnu@0.48.0
+	windows_i686_msvc@0.39.0
+	windows_i686_msvc@0.48.0
+	windows_x86_64_gnu@0.39.0
+	windows_x86_64_gnu@0.48.0
+	windows_x86_64_gnullvm@0.48.0
+	windows_x86_64_msvc@0.39.0
+	windows_x86_64_msvc@0.48.0
+	winnow@0.5.0
+	winres@0.1.12
+	xdg-home@1.0.0
+	yaml-rust@0.4.5
+	zbus@3.14.1
+	zbus_macros@3.14.1
+	zbus_names@2.6.0
+	zvariant@3.15.0
+	zvariant_derive@3.15.0
+	zvariant_utils@1.0.1
+"
+
+inherit cargo
+
+DESCRIPTION="The minimal, blazing-fast, and infinitely customizable prompt for any shell"
+HOMEPAGE="https://starship.rs/"
+SRC_URI="
+	https://github.com/starship/starship/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz
+	${CARGO_CRATE_URIS}
+"
+
+LICENSE="0BSD Apache-2.0 Apache-2.0-with-LLVM-exceptions BSD BSD-2 Boost-1.0 CC0-1.0 ISC MIT MIT-0 MPL-2.0 Unicode-DFS-2016 Unlicense WTFPL-2 ZLIB"
+
+SLOT="0"
+KEYWORDS="~amd64"
+
+BDEPEND="dev-util/cmake
+	>=virtual/rust-1.65"
+
+PATCHES=(
+	# https://bugs.gentoo.org/866133
+	"${FILESDIR}"/${PN}-1.16.0-no-strip.patch
+)
+
+QA_FLAGS_IGNORED="usr/bin/starship"
+
+src_configure() {
+	export PKG_CONFIG_ALLOW_CROSS=1
+	export OPENSSL_NO_VENDOR=true
+
+	cargo_src_configure
+}
+
+src_install() {
+	cargo_src_install
+	dodoc README.md CHANGELOG.md
+}
+
+pkg_postinst() {
+	local v
+	for v in ${REPLACING_VERSIONS}; do
+		if ver_test "${v}" -lt "1.9.0"; then
+			einfo "Note that vicmd_symbol config option was renamed to vimcmd_symbol in version 1.9"
+		fi
+	done
+}
```

Comment:
We can probably get rid of this already?

Reply:
Yeah, I think so, it's an outdated message